src/app/public/modules/description-list/description-list.module.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  CommonModule
+} from '@angular/common';
+
+import {
+  NgModule
+} from '@angular/core';
+
+import {
+  SkyI18nModule
+} from '@skyux/i18n';
+
+import {
+  SkyLayoutResourcesModule
+} from '../shared/layout-resources.module';
+
+import {
+  SkyDescriptionListAdapterService
+} from './description-list-adapter-service';
+
+import {
+  SkyDescriptionListComponent
+} from './description-list.component';
+
+import {
+  SkyDescriptionListContentComponent
+} from './description-list-content.component';
+
+import {
+  SkyDescriptionListTermComponent
+} from './description-list-term.component';
+
+import {
+  SkyDescriptionListDescriptionComponent
+} from './description-list-description.component';
+
+@NgModule({
+  declarations: [
+    SkyDescriptionListComponent,
+    SkyDescriptionListContentComponent,
+    SkyDescriptionListTermComponent,
+    SkyDescriptionListDescriptionComponent
+  ],
+  imports: [
+    CommonModule,
+    SkyI18nModule,
+    SkyLayoutResourcesModule
+  ],
+  exports: [
+    SkyDescriptionListComponent,
+    SkyDescriptionListContentComponent,
+    SkyDescriptionListTermComponent,
+    SkyDescriptionListDescriptionComponent
+  ],
+  entryComponents: [
+    SkyDescriptionListTermComponent,
+    SkyDescriptionListContentComponent
```

Comment:
Curious why these need to be in `entryComponents`?

Reply:
They don't. Leftover from earlier architecture. Removed.